iptables/iptables.go

```diff
@@ -0,0 +1,30 @@
+package iptables
+
+import (
+	"errors"
+	"os"
+
+	"github.com/coreos/go-iptables/iptables"
+)
+
+// AddRule adds the required rule to the host's nat table
+func AddRule(appPort, metadataAddress, hostInterface string) error {
+	hostIP := os.Getenv("HOST_IP")
```

Comment:
I'm using the downward API to provide the pod's IP address as HOST_IP. Agree that using it in a flag is better for consistency. Adding `--host-ip`.